packages/expo-permissions/ios/EXPermissions/EXPermissions.m

```diff
@@ -152,15 +149,9 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
 
 - (NSDictionary *)getPermissionsForResource:(NSString *)type
 {
-  Class requesterClass = _requesters[type];
-  if (requesterClass) {
-    #if __has_include(<EXPermissions/EXRemoteNotificationRequester.h>) || \
-        __has_include(<EXPermissions/EXUserNotificationRequester.h>)
-        if ([requesterClass respondsToSelector:@selector(permissionsWithModuleRegistry:)]) {
-          return [requesterClass permissionsWithModuleRegistry:_moduleRegistry];
-        }
-    #endif
-    return [requesterClass permissions];
+  id<EXPermissionRequester> requester = _requesters[type];
```

Comment:
Here you access this property directly and below you have method named `getPermissionRequesterForType` doing exactly same thing. Rethink it and unify somehow 😉